src/renderers/shaders/ShaderChunk/shadowmap_pars_fragment.glsl.js

```diff
@@ -65,6 +65,27 @@ export default /* glsl */`
 
 	#endif
 
+	float interleavedGradientNoise( vec2 position_screen ) {
+
+		vec3 magic = vec3( 0.06711056, 0.00583715, 52.9829189 );
+		return fract( magic.z * fract( dot( position_screen, magic.xy ) ) );
```

Comment:
Do you think you can use the reference implementation from #32074? The `WebGPURenderer` version of this PR can use the existing utility function then. The code from #32074 is also the "official" one from Jimenez 2014.

Reply:
Done!